applications/openshift/kubelet/kubelet_eviction_thresholds_set_hard_imagefs_available/kubernetes/shared.yml

```diff
@@ -1,5 +1,9 @@
 ---
 # platform = multi_platform_ocp
-{{{ kubelet_config(path='kubeletConfig.evictionHard',parameter='imagefs.available', value='var_kubelet_evictionhard_imagefs_available') }}}
+{{{ kubelet_config(path='kubeletConfig.evictionHard',parameter='imagefs.available', value='var_kubelet_evictionhard_imagefs_available', role='var_role_worker') }}}
 ---
-{{{ kubelet_config_fixed(path='kubeletConfig',parameter='evictionPressureTransitionPeriod', value='0s') }}}
\ No newline at end of file
+{{{ kubelet_config_fixed(path='kubeletConfig',parameter='evictionPressureTransitionPeriod', value='0s', role='var_role_worker') }}}
```

Comment:
I see that it was the case also before but why does the remediation include also the `evictionPressureTransitionPeriod` even if the rule doesn't check for it? Is it a dependency?

Reply:
I think it was because the remediation fails if that is not specific, but I am not sure if that is still the case